security/aporeto/docs/README.md

```diff
@@ -0,0 +1,337 @@
+# Introduction
+
+This tutorial will guide you through setting up custom security policy within your OpenShift Container Platform (OCP) namespace, between namespaces, or from your namespace to an external system provided it meets certain criteria.
+
+## Trust No One
+
+With the addition of new security components Platform Services is able to offer product teams the ability to operate in a Zero Trust model; In a Zero Trust is a security model you don't trust anything outside of your own components (pods).
+
+This is done by way of creating application identities for each component (Web, API, Database, etc) and based on this identity allowing specific components to talk to one another by creating `NetworkSecurityPolicy` (NSP) or `ExternalNetworks` (EN).
+
+**🤓 ProTip**
+
+* Assume that the network and platform are insecure and built up robust security practices.
+
+## How it Works
```

Comment:
Would be great to have "Step 1", "Step 2", etc added to the guide section titles.

Reply:
I'll add a ToC so people can quickly navigate to different sections. I think I like it as is though.